github/ci/prow-deploy/kustom/base/manifests/local/prow-rehearse-deployment.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
         - name: rehearse
-          image: quay.io/kubevirtci/rehearse:v20210312-b3a5e0f2
+          image: docker.io/kubevirtci/rehearse:v20200923-8930f83
```

Comment:
Why do we use docker again? Also this image seems very old?

Reply:
Indeed, this is the one we were using in the old prow control plane. 

The latest on quay doesn't trigger the jobs for some reason, see https://github.com/kubevirt/project-infra/pull/851#issuecomment-828498877 

IIRC we tried to bump to the latest rehearse image on quay when we fixed the tests and changed the build process to bazel but it had also this issue of not triggering jobs, then we reverted to the older one.

Reply:
Lets mirror it to quay ?
since we don't have proxy atm on the new cluster, so we might hit docker rate limit
(even in the future when we do have, a cache miss is prone for that)

Reply:
rehearse plugin runs with prow control plane, we have docker proxy there. I think we should instead try to fix the latest version.